include/seahorn/Support/SeaLog.hh

```diff
@@ -16,6 +16,7 @@
   ::seahorn::warn_ostream(llvm::errs())                                        \
       .prefix("Info: ")                                                        \
       .suffix(FLOC(__FILE__, __LINE__))
+#ifndef DOCTEST_LIBRARY_INCLUDED
```

Comment:
why would we want strange dependence on doctest inside our code?

seems like a wrong way to fix this problem. Just 'undef INFO` in unit tests before doctest is included.

Reply:
done